### ISSUE TITLE
Can't nsenter into mount namespace created by bwrap

### ISSUE THREAD

**Issue by pmakholm** [projectatomic/bubblewrap#308]
I am using bwrap to set up a restricted mount namespace

```
pmakholm@pmakholm:~$ echo $$
11706
pmakholm@pmakholm:~$ bwrap --ro-bind / / --tmpfs /home/pmakholm /bin/bash
pmakholm@pmakholm:~$ echo $$
11860
pmakholm@pmakholm:~$ ls -l /bin/bash
-rwxr-xr-x 1 root root 1099016 May 15  2017 /bin/bash
pmakholm@pmakholm:~$ 
```

I then try to enter the same namespace using `nsetner`, but I get an error

```
pmakholm@pmakholm:~$ sudo nsenter --target 11860 -m /bin/bash
[sudo] password for pmakholm: 
nsenter: failed to execute /bin/bash: No such file or directory
pmakholm@pmakholm:~$ 
```

Setting up a namespace using unshare and pivot_root by hand allows me to enter the resulting mountnamespace.

**Comment by smcv** [comment]
Which version of bwrap is this, and are you using the setuid or unprivileged code path?

I can't reproduce this with Debian's bwrap 0.3.1-4, using the setuid code path.

**Comment by pmakholm** [comment]
Ohhh, I had misplaced the sources.list entry for stretch-backport so I used the version included with stretch (0.1.7). I can't reproduce it in 0.3.1 included in stretch-backport.

So, closing due to me using an obsolete version. Thanks for your help.

**Comment by smcv** [comment]
My best guess is that this was fixed by #256.
